app/models/user.rb

```diff
@@ -0,0 +1,7 @@
+class User < ApplicationRecord
+    has_many :posts, dependent: :destroy
```

Comment:
**You did a great job! Almost there**

- [ ] There are places that we need to work on. Let me show you the places :) For starting, the user has many posts as well as comments and likes. Kindly add them as well :)

It would be nice if you could add foreign_key: 
```
has_many :comments, foreign_key: 'author_id'
```
for all has many connections in user.